### DISCUSSION TITLE
update frontend

### DISCUSSION THREAD

**Discussion by turbopixel** [YOURLS/YOURLS#3169] [Ideas]
Hi, YOURLS is a really great tool!

What do you think about a new frontend? I work full time as a frontend developer and prefer to renew existing frontends. YOURLS offers me a good basis. 

What do you think about css frameworks (bulma or bootstrap) or a completely own design? 

I am ready to pour my free time into this project.

**Comment by LeoColomb** [comment]
Thanks a lot for your comment @turbopixel. 
There is a v2 under active development (not public yet) which is coming with a brand new web interface, with Tailwind as base framework. 
It will be great to get your input when released!
